aries_cloudagent/vc/vc_ld/models/credential.py

```diff
@@ -341,6 +349,13 @@ class Meta:
         metadata={"example": CREDENTIAL_SUBJECT_EXAMPLE},
     )
 
+    credential_status = DictOrDictListField(
+        required=False,
+        data_key="credentialStatus",
+        # validate=CREDENTIAL_STATUS_VALIDATE,
+        # metadata={"example": CREDENTIAL_STATUS_EXAMPLE},
```

Comment:
General recommendation to not include commented out code. If these are intended to represent a future `TODO` item, I'd replace the commented lines with a `# TODO add validation and example`.

Reply:
+1